packages/flare/tests/bin/test_data_store.py

```diff
@@ -0,0 +1,26 @@
+from data_store import ConfigDataStore
+from datetime import datetime
+
+
+def test_get_and_set_last_tenant_id(data_store: ConfigDataStore) -> None:
+    data_store.set_last_tenant_id(123)
+    assert data_store.get_last_tenant_id() == 123
+
+
+def test_get_and_set_start_date(data_store: ConfigDataStore) -> None:
+    date = datetime(2024, 3, 6, 12, 0, 0)
```

Comment:
Oh no this is an evil datetime without timezone!

I guess its fine here since its a test, but these dates are very dangerous because they will cause crashes when compared with timezoned dates.